### PR TITLE
Add support of MultiPolygon and GeometryCollection to 'orient' function

### PR DESCRIPTION
**Pull request by drnextgis**
Fix for https://github.com/Toblerity/Shapely/issues/635.

### CODE REVIEW

File: shapely/geometry/polygon.py [hunk 1]
```diff
@@ -388,22 +387,6 @@ def asPolygon(shell, holes=None):
...
-    else:
-        rings.append(list(ring.coords)[::-1])
-    for ring in polygon.interiors:
-        if signed_area(ring)/s <= 0.0:
-            rings.append(ring)
-        else:
-            rings.append(list(ring.coords)[::-1])
-    return Polygon(rings[0], rings[1:])
-
-
```
Comment (sgillies) [inline shapely/geometry/polygon.py]: If we `from shapely.ops import orient` in polygon.py, we won't break any code that depends on importing it from here.

Comment (drnextgis) [reply shapely/geometry/polygon.py]: Actually adding that import causes circular dependency issue. I've implemented your suggestion in a little different way.

**Comment by pglotov** [comment]
Shouldn't it've been added to the docs?

**Comment by sgillies** [comment]
Docs added in 053df24.

**Comment by vpipkt** [comment]
@drnextgis This saved me a ton of headache today! 🍰

**Pull request merged**